src/app/shared/interfaces/metadata.ts

```diff
@@ -0,0 +1,22 @@
+export interface MetaData {
+  checked: boolean;
+  correlationId: string;
+  duration: string;
+  endTime: string;
+  estimatedMemoryUsage: string;
+  name: string;
+  numberOfCheckpoints: string;
+  status: string;
+  storageId: string;
+  storageSize: string;
+
+  COMPONENT: string;
+  'CONVERSATION ID': string;
+  'CORRELATION ID': string;
+  'ENDPOINT NAME': string;
+  'NR OF CHECKPOINTS': string;
+  STATUS: string;
+  TIMESTAMP: string;
+
+  [key: string]: string | boolean;
```

Comment:
I feel like this is not necessary for an object thats retrieved from the backend. This should only be used when you do not know what the object can contain.

Reply:
This has been taken out.